File: app/src/main/java/com/example/pam_project/features/tasks/edit/EditTaskView.java
package com.example.pam_project.features.tasks.edit;

import com.example.pam_project.features.tasks.list.TaskInformation;

public interface EditTaskView {

    void bindTask(TaskInformation model);

    void onTaskDelete();

    void onTaskUpdatedError();

    void onTaskDeletedError();
    
    void showDeleteDialog();

    void onTaskRetrievedError();
}

File: app/src/main/java/com/example/pam_project/features/tasks/edit/EditTaskView.kt
package com.example.pam_project.features.tasks.edit

import com.example.pam_project.features.tasks.list.TaskInformation

interface EditTaskView {
    fun bindTask(model: TaskInformation?)
    fun onTaskDelete()
    fun onTaskUpdatedError()
    fun onTaskDeletedError()
    fun showDeleteDialog()
    fun onTaskRetrievedError()
}